### PR TITLE
Minor tweaks to Spanish translations

### PR DESCRIPTION
**Pull request by joseluistorres**
Minor tweaks for the Spanish translation.
﻿

### Pull request checklist

- [x] Your code contains tests relevant for code you modified
- [] You have linted and tested the project with `bin/check`

### CODE REVIEW

File: config/locales/es.yml [hunk 1]
```diff
@@ -180,7 +180,7 @@ es:
       subtitle_1: railsdevs
       subtitle_2: empodera a los desarrolladores independientes disponibles para su próximo trabajo.
       subtitle_3: Deja de buscar en las bolsas de trabajo y relájate mientras las empresas se comunican contigo.
-      title_1: La bolsa de trabajo invertida para
+      title_1: La bolsa de trabajo enfocada para
```
Comment (salochara) [inline config/locales/es.yml:183]: Hmm 🤔 
I left the "invertida" part to keep the fact of railsdevs being a reverse job board. I feel that we'd lose that with "enfocada". What do you think? @joseluistorres 
cc @joemasilotti

**Comment by salochara** [comment]
Thanks for your help on this one @joseluistorres!

File: config/locales/devise.views.es.yml [hunk 2]
```diff
@@ -120,7 +120,7 @@ es:
         sign_in_with_provider: Iniciar sesión con %{provider}
         sign_up: Registrarse
       minimum_password_length:
-        one: "(%{count} caractere como mínimo)"
+        one: "(%{count} caracter como mínimo)"
```
Comment (salochara) [inline config/locales/devise.views.es.yml:123]: Good catch! 
This is actually taken from the [devise locale](https://github.com/tigrish/devise-i18n/blob/master/rails/locales/es.yml). Maybe you can propose that change for the gem?

**Comment by joseluistorres** [comment]
Of course! Trabajo con varios michelados

**Comment by salochara** [comment]
Que chido! En donde andas trabajando ahorita?

**Comment by joseluistorres** [comment]
Yeah I didn't get that right, invertida makes sense with that context though 👍On Jan 30, 2022 8:46 PM, Salomón Charabati ***@***.***> wrote:

@salochara commented on this pull request.







In config/locales/es.yml:

> @@ -180,7 +180,7 @@ es:

       subtitle_1: railsdevs

       subtitle_2: empodera a los desarrolladores independientes disponibles para su próximo trabajo.

       subtitle_3: Deja de buscar en las bolsas de trabajo y relájate mientras las empresas se comunican contigo.

-      title_1: La bolsa de trabajo invertida para

+      title_1: La bolsa de trabajo enfocada para



Hmm 🤔

I left the "invertida" part to keep the fact of railsdevs being a reverse job board. I feel that we'd lose that with "enfocada". What do you think? @joseluistorres

cc @joemasilotti

File: config/locales/devise.views.es.yml (hunk 2 above)
Comment (joseluistorres) [reply config/locales/devise.views.es.yml:123]: That's bad Spanish though, it doesn't matter if they wrote it 😉

Comment (salochara) [reply config/locales/devise.views.es.yml:123]: Yeah I agree! 
I meant to also suggest the change to the [gem](https://app.localeapp.com/projects/377/). I've done the suggestion already 😀

**Comment by joemasilotti** [comment]
I'm going to have to take your word on this one :)

Thanks for the contribution @joseluistorres and double checking @salochara!

**Pull request merged**